### PR TITLE
OP-264: Fix display names for enquire values

### PR DESCRIPTION
**Pull request by dborowiecki**
TICKET: [OP-264](https://openimis.atlassian.net/browse/OP-264)

### CODE REVIEW

File: app/src/main/java/org/openimis/imispolicies/Enquire.java [hunk 1]
```diff
@@ -393,8 +393,8 @@ public void run() {
                              TotalDelivieriesLeft = jsonObject.getString("totalDelivieriesLeft").equalsIgnoreCase("null")? "": "TotalDelivieriesLeft: "+jsonObject.getString("totalDelivieriesLeft");
```
Comment (dragos-dobre) [inline app/src/main/java/org/openimis/imispolicies/Enquire.java:393]: Fix spell: totalDeliveriesLeft

Comment (dragos-dobre) [reply app/src/main/java/org/openimis/imispolicies/Enquire.java:393]: "TotalDelivieriesLeft: " is hardcoded. Please add this in strings.xml.

File: app/src/main/java/org/openimis/imispolicies/Enquire.java [hunk 2]
```diff
@@ -393,8 +393,8 @@ public void run() {
...
                              TotalAntenatalLeft = jsonObject.getString("totalAntenatalLeft").equalsIgnoreCase("null")? "": "TotalAntenatalLeft: "+jsonObject.getString("totalAntenatalLeft");
```
Comment (dragos-dobre) [inline app/src/main/java/org/openimis/imispolicies/Enquire.java:394]: "TotalAntenatalLeft: " is hardcoded. Please add this in strings.xml.

File: app/src/main/java/org/openimis/imispolicies/Enquire.java [hunk 3]
```diff
@@ -393,8 +393,8 @@ public void run() {
...
                              ConsultationAmountLeft = jsonObject.getString("consultationAmountLeft").equalsIgnoreCase("null")? "": "ConsultationAmountLeft: "+jsonObject.getString("consultationAmountLeft");
```
Comment (dragos-dobre) [inline app/src/main/java/org/openimis/imispolicies/Enquire.java:395]: "ConsultationAmountLeft: " is hardcoded. Please add this in strings.xml.

File: app/src/main/java/org/openimis/imispolicies/Enquire.java [hunk 4]
```diff
@@ -393,8 +393,8 @@ public void run() {
...
-                             SurgeryAmountLeft = jsonObject.getString("surgeryAmountLeft").equalsIgnoreCase("null")? "": "TotalAdmissionsLeft: "+jsonObject.getString("surgeryAmountLeft");
-                             HospitalizationAmountLeft = jsonObject.getString("hospitalizationAmountLeft").equalsIgnoreCase("null")? "": "TotalAdmissionsLeft: "+jsonObject.getString("hospitalizationAmountLeft");
+                             SurgeryAmountLeft = jsonObject.getString("surgeryAmountLeft").equalsIgnoreCase("null")? "": "SurgeryAmountLeft: "+jsonObject.getString("surgeryAmountLeft");
```
Comment (dragos-dobre) [inline app/src/main/java/org/openimis/imispolicies/Enquire.java:396]: "SurgeryAmountLeft: " is hardcoded. Please add this in strings.xml.

File: app/src/main/java/org/openimis/imispolicies/Enquire.java [hunk 5]
```diff
@@ -393,8 +393,8 @@ public void run() {
...
+                             HospitalizationAmountLeft = jsonObject.getString("hospitalizationAmountLeft").equalsIgnoreCase("null")? "": "HospitalizationAmountLeft: "+jsonObject.getString("hospitalizationAmountLeft");
```
Comment (dragos-dobre) [inline app/src/main/java/org/openimis/imispolicies/Enquire.java:397]: "HospitalizationAmountLeft: " is hardcoded. Please add this in strings.xml.

File: app/src/main/java/org/openimis/imispolicies/Enquire.java [hunk 6]
```diff
@@ -393,8 +393,8 @@ public void run() {
...
                              AntenatalAmountLeft = jsonObject.getString("antenatalAmountLeft").equalsIgnoreCase("null")? "": "AntenatalAmountLeft: "+jsonObject.getString("antenatalAmountLeft");
```
Comment (dragos-dobre) [inline app/src/main/java/org/openimis/imispolicies/Enquire.java:398]: "AntenatalAmountLeft: " is hardcoded. Please add this in strings.xml.

**Review by dragos-dobre — CHANGES_REQUESTED** [review]

File: app/src/main/java/org/openimis/imispolicies/Enquire.java (hunk 1 above)
Comment (dborowiecki) [reply app/src/main/java/org/openimis/imispolicies/Enquire.java]: Both tblControls and endpoint use "totalDelivieriesLeft" key, so I kept it there but removed typo from the display. Hardcoded string moved to strings.xml

File: app/src/main/java/org/openimis/imispolicies/Enquire.java (hunk 2 above)
Comment (dborowiecki) [reply app/src/main/java/org/openimis/imispolicies/Enquire.java]: Fixed

File: app/src/main/java/org/openimis/imispolicies/Enquire.java (hunk 3 above)
Comment (dborowiecki) [reply app/src/main/java/org/openimis/imispolicies/Enquire.java]: Fixed

File: app/src/main/java/org/openimis/imispolicies/Enquire.java (hunk 4 above)
Comment (dborowiecki) [reply app/src/main/java/org/openimis/imispolicies/Enquire.java]: Fixed

File: app/src/main/java/org/openimis/imispolicies/Enquire.java (hunk 5 above)
Comment (dborowiecki) [reply app/src/main/java/org/openimis/imispolicies/Enquire.java]: Fixed

File: app/src/main/java/org/openimis/imispolicies/Enquire.java (hunk 6 above)
Comment (dborowiecki) [reply app/src/main/java/org/openimis/imispolicies/Enquire.java]: Fixed

**Comment by dborowiecki** [comment]
@dragos-dobre I added requested changes, strings were moved to resources xml, but I put same english translation in english, french and swahili resources. Should we also update lokalise dictionaries so they also include new keys?

**Review by dragos-dobre — APPROVED** [review]

**Comment by dragos-dobre** [comment]
I added the keys in Lokalise.

**Pull request merged**